simulation/src/main/java/com/palantir/dialogue/core/SimulationMetricsReporter.java

```diff
@@ -116,17 +135,39 @@ public XYChart chart(Pattern metricNameRegex) {
         chart.getStyler().setToolTipsEnabled(true);
         chart.getStyler().setToolTipsAlwaysVisible(true);
 
+        if (!simulation.events().getEvents().isEmpty()) {
+            double[] eventXs = simulation.events().getEvents().keySet().stream()
+                    .mapToDouble(nanos -> TimeUnit.NANOSECONDS.toMillis(nanos) / 1000d)
```

Comment:
Is this equivalent to:
```suggestion
                    .mapToDouble(nanos -> nanos / 1_000_000_000D)
```
It took me a moment to parse that we were converting nanos to seconds with decimals.

Reply:
This is copy pasta from what was already there, I did not even read this code, just shuffled things around.

Reply:
Wait, isn't this just trying to convert to seconds? but just like get fractional seconds.

Reply:
Seems to not change the pngs, so let's do it. I also extracted a method to give this a name.